frontend/src/components/AnnotationView/AnnotationSidebar.vue

```diff
@@ -30,7 +30,10 @@ div {
   padding: var(--p-8);
   margin-top: 0.625rem;
   margin-left: 0.625rem;
-  margin-bottom: 0.625rem
+  margin-bottom: 0.625rem;
```

Comment:
This is the perfect place to go ahead and fix the sizing to use the CSS design system spacing we have in place.  `0.625rem` is `var(--p-10)` in our case.  Your additional bottom margin plus fixing the old values here for the margin top/left/bottom to use `--p-10` are a good example of cleaning up some CSS as we go.

Reply:
I'll be sure to double-check in the future if we already have a variable setup for this. I see it in main.css now

The change has been made